packages/terra-icon/src/icon/IconAlertLowLight.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import classNamesBind from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
 import IconBase from '../IconBase';
-import styles from '../IconAlertLowLight.module.scss';
+import styles from '../clinical-lowlight-theme/IconAlert.module.scss';
```

Comment:
this change doesn't seem right, we shouldn't be pulling the lowlight theme directly